tools/xtb/xtb_molecular_optimization.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="xtb_molecular_optimization" name="xtb molecular optimization" version="@TOOL_VERSION@+galaxy0" profile="21.09">
+    <description>Semiempirical quantum mechanical molecular optimization method</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">xtb</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln '${coordinates}' 'mol.${coordinates.ext}' &&
```

Comment:
ln -s  softlinks, not hardlinks

Reply:
Thanks for your invaluable linux skills which I am so dearly lacking!